src/libsync/clientsideencryption.cpp

```diff
@@ -298,7 +298,7 @@ namespace {
     };
 
     QByteArray BIO2ByteArray(Bio &b) {
-        int pending = BIO_ctrl_pending(b);
+        int pending = static_cast<int>(BIO_ctrl_pending(b));
```

Comment:
please use `auto pending` to not repeat `int` twice

Reply:
done